pkg/content/content.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package content
+
+import (
+	"github.com/GoogleContainerTools/kpt/pkg/content/extensions"
+)
+
+type Content = extensions.Content
```

Comment:
Because the `pkg/content` code needs to see a few go pkg which need to accept `content.Content` as argument.

The alias enables `content.Content` and `content.EverythingElse()` to be in one place from the consumer's PoV, but in the places where that caused a cycle the transitive location could use `extensions.Content` leaf location for type definitions.

TBH, it originally contained only the duck-typing interfaces but `content.Content` was eventually was closed back over as well.